packages/store/src/devicesSlice/selectors.ts

```diff
@@ -87,6 +87,18 @@ export const getLimitMaxed = createSelector(
   ({ total: acceptedDevices = 0 }, combinedLimit) => combinedLimit <= acceptedDevices
 );
 
+export const getAcceptedTiers = createSelector([getDevicesById], devices =>
+  Object.values(devices)
+    .filter(device => device.status === 'accepted')
+    .reduce(
+      (acc, curr) => {
+        acc[curr.tier || 'standard']++;
```

Comment:
I'd also checked if acc[curr.tier] is not undefined, because backend might return new tier that is not listed in  { micro: 0, system: 0, standard: 0 }